### DISCUSSION TITLE
Call to a member function map() on null

### DISCUSSION THREAD

**Discussion by pit07** [spatie/laravel-tags#409] [Q&A]
Hi everybody!

I have some issues with this package, i don't understand what I'm doing wrong.

I installed package correctly, tables are created.
My model:

```
namespace App\Models;

use Spatie\Tags\HasTags;

class Product extends Model implements HasMedia
{
    use HasTags;
```

And in my Nova file:

```
Tags::make('Tags', 'tags'),
```
I have this error :

**Call to a member function map() on null** on Edit Page.

The taggables and tags tables are ok:

![image](https://user-images.githubusercontent.com/19688070/173832684-60c0069e-50f9-4c25-a103-b668535e101e.png)

![image](https://user-images.githubusercontent.com/19688070/173832717-40c7bcd5-1d08-4ded-a984-72da6cc3b9bb.png)

![image](https://user-images.githubusercontent.com/19688070/173833804-2ec24eae-de15-4820-bcee-b518b980ad06.png)

Any ideas?
Thanks a lot !

**Comment by nevstas** [comment]
You can not use "Laravel\Nova\Fields\Tag".
Because relation should be "BelongsToMany".
Use "MorphToMany" instead of "BelongsToMany".
MorphToMany::make('Tags'),
